Workbooks/AKS/Syslog/Syslog.workbook

```diff
@@ -1,568 +1,561 @@
+						"quote": "'",
+						"delimiter": ",",
+						"query": "let T = (Syslog | distinct Facility );\r\nlet T_has_records = toscalar(Syslog | summarize count() > 0);\r\nunion \r\n(T | where T_has_records == true), (datatable(Facility:string)['Facility'] | where T_has_records == false)\r\n| distinct Facility",
+						"crossComponentResources": [
+							"{Workspace}"
+						],
+						"typeSettings": {
+							"additionalResourceOptions": [
+								"value::all"
+							],
+							"showDefault": false
+						},
+						"defaultValue": "value::all",
+						"queryType": 0,
+						"resourceType": "microsoft.operationalinsights/workspaces"
+					},
+					{
+						"id": "e73d71c1-435b-4916-a2c8-25dabbdefa45",
+						"version": "KqlParameterItem/1.0",
+						"name": "SeverityLevel",
+						"type": 2,
+						"isRequired": true,
+						"multiSelect": true,
+						"quote": "'",
+						"delimiter": ",",
+						"query": "let T = (Syslog | distinct SeverityLevel );\r\nlet T_has_records = toscalar(Syslog | summarize count() > 0);\r\nunion \r\n(T | where T_has_records == true), (datatable(SeverityLevel:string)['SeverityLevel'] | where T_has_records == false)\r\n| distinct SeverityLevel",
+						"crossComponentResources": [
+							"{Workspace}"
+						],
+						"typeSettings": {
+							"additionalResourceOptions": [
+								"value::all"
+							],
+							"showDefault": false
+						},
+						"defaultValue": "value::all",
+						"queryType": 0,
+						"resourceType": "microsoft.operationalinsights/workspaces",
+						"value": [
+							"value::all"
+						]
+					}
+				],
+				"style": "pills",
+				"queryType": 0,
+				"resourceType": "microsoft.operationalinsights/workspaces"
+			},
+			"name": "Parameters"
+		},
+		{
+			"type": 1,
+			"content": {
+				"json": "# Syslog\r\n\r\n\r\n\r\n"
+			},
+			"name": "Title text"
+		},
+		{
+			"type": 3,
+			"content": {
+				"version": "KqlItem/1.0",
+				"query": "Syslog\r\n| where HostName in ({HostName}) or '*' in ({HostName})\r\n| where Facility in ({Facility}) or '*' in ({Facility})\r\n| where SeverityLevel in ({SeverityLevel}) or '*' in ({SeverityLevel})\r\n| summarize count(SeverityLevel) by SeverityLevel \r\n| extend jkey = 1\r\n| join (Syslog\r\n| where HostName in ({HostName}) or '*' in ({HostName})\r\n| where Facility in ({Facility}) or '*' in ({Facility})\r\n| where SeverityLevel in ({SeverityLevel}) or '*' in ({SeverityLevel})\r\n  | make-series Trend = count(SeverityLevel) default = 0 on TimeGenerated from {timeRange:start} to {timeRange:end} step {timeRange:grain} by SeverityLevel) on SeverityLevel",
+				"size": 4,
+				"title": "Summary by Severity Level",
+				"noDataMessage": "No results returned - Enable Syslog or extend time-range",
+				"noDataMessageStyle": 5,
+				"timeContextFromParameter": "timeRange",
+				"queryType": 0,
+				"resourceType": "microsoft.operationalinsights/workspaces",
+				"crossComponentResources": [
+					"{Workspace}"
+				],
+				"visualization": "tiles",

```

Comment:
Can we make this tile unclickable?

Reply:
Removed the border so it doesn't like a button now. Any better way to handle?